Authentication/AuthenticationBootstrapper.swift

```diff
@@ -63,6 +62,13 @@ public class AuthenticationBootstrapper<User: UserType, SessionService: SessionS
             }
         }
     }
+    
+    public convenience init(sessionService: SessionService, window: UIWindow, termsAndServicesURL: NSURL,
+                            mainViewControllerFactory: () -> UIViewController) {
+        self.init(sessionService: sessionService, window: window,
+                  viewConfiguration: AuthenticationViewConfiguration(termsAndServicesURL: termsAndServicesURL),
+                  mainViewControllerFactory: mainViewControllerFactory)
+    }
```

Comment:
Because there is no possible default terms and services URL, so now there is no default `SignupViewConfiguration` and so there is no default `AuthenticationViewConfiguration`. But the most basic `AuthenticationViewConfiguration` receives an url, so I created this convenience init to make it easier for the user to create the bootstrapper with the url without having to create the `AuthenticationViewConfiguration`.